src/main/java/org/sagebionetworks/web/client/transform/JSONEntityFactoryImpl.java

```diff
@@ -18,13 +19,14 @@
 public class JSONEntityFactoryImpl implements JSONEntityFactory {
 	// This factory allows the creation of new class instances using only the class name.
 	// Since reflection is not allowed in GWT, it uses an auto-generated swtich.
-	AutoGenFactory internalFactory;
+	AutoGenFactory internalFactory = null;

```

Comment:
need to remove all uses of PaginatedResults, or remove autogen factory from PaginatedResults